frontend/src/modules/app/layouts/index.ts

```diff
@@ -1,3 +1,9 @@
+/**
+ * @summary A short one-line description for the file
```

Comment:
These placeholders need to be filled in. I would suggest including a description of how and/or where the layout files in this folder are combined with other files specifying the contents inside the layouts. For example, if we need to define pages with a new layout, what files should we modify before or after creating a new layout file in this folder?

Reply:
I am adding the following to the docs
```
/**
 * @summary Components compreising of all Layout files
 * These Layout files are used in `Route.tsx` file 
 * to define which layout to use for which page route
 *
 * @file Layout file contains wrapper for Sidebar & Toolbar
 * To create new customized layout will require
 * to add a file in this directory with customized changes
 * 
 * To use the layout for a page, Define Layout while declaring page route, Example below
 * `SidebarSlideRoute` layout can be used by `NewPage` using following example
 * `<SidebarSlideRoute exact path={NewRoutePath} component={NewPage} />`
 */
```